ol_mrp_plm_tier_validation/tests/test_tier_validation.py

```diff
@@ -1,48 +1,73 @@
 # Import Odoo libs
 from odoo.tests import common, tagged
+from odoo.exceptions import ValidationError
 
 
 @tagged("-at_install", "post_install")
 class TestProductTierValidation(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        group_ids = cls.env.ref("base.group_system").ids
-        cls.test_user_1 = cls.env["res.users"].create(
-            {
-                "name": "John",
-                "login": "test1",
-                "groups_id": [(6, 0, group_ids)],
-                "email": "test@examlple.com",
-            }
-        )
-
-        cls.tier_def_obj = cls.env["tier.definition"]
+        # Set variables from demo data
+        cls.test_user_1 = cls.env.ref("base.partner_admin")
         cls.eco_type = cls.env.ref("mrp_plm.ecotype0")
         cls.product = cls.env.ref(
             "mrp.product_product_computer_desk_bolt_product_template"
         )
+        # Set states on the stages which tier validation uses
         cls.draft_stage = cls.env.ref("mrp_plm.ecostage_new").write(
```

Comment:
@patrickrwilson Code should be.. please change other accordingly.

cls.draft_stage = cls.env.ref("mrp_plm.ecostage_new")
cls.draft_stage.write(
            {"state": "progress"}
        )